deeppavlov/configs/ner/ner_mbert_dream_distil.json

```
@@ -0,0 +1,155 @@
+{
+    "dataset_reader": {
+        "class_name": "conll2003_reader",
+        "dataset_name": "conll2003",
+        "provide_pos": false
+    },
+    "dataset_iterator": {
+        "class_name": "data_learning_iterator"
+    },
+    "chainer": {
+        "in": [
+            "x"
+        ],
```

```suggestion
        "in": ["x"],
```
And edit other lists where newline isn't necessary. I understand that this is automatic json formatting, but it's less readable than incorrect formatting.